core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/iterator/package-info.java

```diff
@@ -1,15 +1,22 @@
 /*******************************************************************************
- * Copyright (c) 2019 Eclipse RDF4J contributors. All rights reserved. This program and the accompanying materials are
- * made available under the terms of the Eclipse Distribution License v1.0 which accompanies this distribution, and is
- * available at http://www.eclipse.org/org/documents/edl-v10.php.
- ******************************************************************************/
-@InternalUseOnly
+ * Copyright (c) 2019 Eclipse RDF4J contributors.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *******************************************************************************/
+
 /**
  * Implementations of {@link Iteration} relevant to query evaluation.
  *
  * @apiNote This feature is for internal use only: its existence, signature or behavior may change without warning from
  *          one release to the next.
  */
+
+@InternalUseOnly
```

Comment:
@jeenbroekstra the maven formatter is configured to not format the header comment, but it doesn't recognise the copyright header as the header comment if the `@InternalUseOnly` annotation comes in between the copyright header and the javadoc comment. I'm not sure why. My fix is to just move the annotation.